### ISSUE TITLE
Board Review: Introducing  Application Insights OpenTelemtery distro

### ISSUE THREAD

**Issue by lmolkova** [Azure/azure-sdk#5422]
## Contacts and Timeline

* Service team responsible for the client library: **Azure Monitor (ApplicationInsights)**
* Main contacts: **Matthew McCleary**, **Liudmila Molkova** (@lmolkova), **Leighton Chen** (@lzchen)
* Expected stable release date for this library: April 2023 (TBD)

## About the Service 

* Link to documentation introducing/describing the service: https://learn.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview
* Link to the service REST APIs: **N/A**
* Is the goal to release a Public Preview, Private Preview, or GA?  **Public preview and then GA**

## About the client library

Azure Monitor is working on OpenTelemtery distros -  uber-packages that bring and configure all users need to instrument their applications (OpenTelemetry, instrumentation libraries, and exporter) to send all kinds of telemetry to ApplicationInsights.

The plan is that only Python will have green-field SDK that's based on vanilla OpenTelemetry. Other languages will evolve existing Application Insights SDKs adding new OTel-based experiences to them for the time being.

* Name of client library:
  * Python: **azure-monitor-applicationinsights-opentelemetry** 
  *  Future (not creating green-filled packages for now, listing to demonstrate the principle):
     *  .NET: `Azure.Monitor.ApplicationInsights.OpenTelemetry.AspNetCore` or  `Azure.Monitor.ApplicationInsights.OpenTelemetry.WorkerService`
* Link to library reference documentation:
  * https://github.com/microsoft/ApplicationInsights-Python/tree/main/azure-monitor-opentelemetry-distro
* Is there an existing SDK library? If yes, provide link: 
  * https://github.com/microsoft/ApplicationInsights-Python/tree/main/azure-monitor-opentelemetry-distro

## Step 1: Champion Scenarios 

* Minimal distro configuration for traces, metrics, and logs
    * TODO
* configuring tracing
    * https://github.com/microsoft/ApplicationInsights-Python/blob/main/azure-monitor-opentelemetry-distro/samples/tracing/simple.py
    * TBD: sampling?
* configuring metrics
  * TODO
* configuring logs
    * https://github.com/microsoft/ApplicationInsights-Python/blob/main/azure-monitor-opentelemetry-distro/samples/logging/simple.py

TODO: API view

## Step 2: Quickstart Samples (Optional)

```python
from azure.monitor.opentelemetry.distro import configure_azure_monitor
from opentelemetry import trace

configure_azure_monitor(
    connection_string="<your-connection-string>",
    service_name="foo_service",
    tracing_export_interval_millis=15000,
    logging_level=WARN
)

logger = getLogger(__name__)
tracer = trace.get_tracer(__name__)

with tracer.start_as_current_span("hello"):
    print("Hello, World!")
    logger.info("Correlated info log")
    logger.warning("Correlated warning log")
    logger.error("Correlated error log")
```

**Comment by lmolkova** [comment]
closed in favor #5423

**Comment by lzchen** [comment]
@lmolkova 

Should I refer to this issue or https://github.com/Azure/azure-sdk/issues/5423 ? Can this issue be closed?
